usr/sbin/p11sak/p11sak.c

```diff
@@ -2588,6 +2814,31 @@ static CK_RV close_session(CK_SESSION_HANDLE session)
     return rc;
 }
 
+static CK_RV parse_file() {
+    CK_RV rc = CKR_OK;
+    FILE *fp;
+    char *file_loc = getenv("P11SAK_DEFAULT_CONF_FILE");
+
+    if (!file_loc) {
+        file_loc = P11SAK_DEFAULT_CONF_FILE;
+    } 
```

Comment:
brackets not needed for a single line body.

Reply:
Done